addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -41,7 +38,9 @@ options.registry.SnippetPopup = options.Class.extend({
      */
     moveBlock: function (previewMode, widgetValue, params) {
         const $container = $(widgetValue === 'moveToFooter' ? 'footer' : 'main');
-        this.$target.closest('.s_popup').prependTo($container.find(':o_editable').first());
+        this.$target.closest('.s_popup').prependTo(
+            $container.find(':o_editable:not(.o_snippet_invisible):not(.o_snippet_invisible *)').first()
```

Comment:
Same comment for this commit as the first one but here I would remove the commit, or adapt it here to `.oe_structure:o_editable` maybe ? And the option should be hidden if no such element exists ?

Reply:
Actually, the revert commit you mentionned did not fix this one.
I fixed/avoided the issue by inserting the cookies template (inside instead of before div[id=footer]).
I squashed this one with the previous one and used `.oe_structure:o_editable` for a more generic safety nest.